tests_e2e/test_suites/agent_update.yml

```diff
@@ -2,5 +2,8 @@ name: "AgentUpdate"
 tests:
   - "agent_update/rsm_update.py"
 images: "endorsed"
-location: "eastus2euap"
-owns_vm: true
\ No newline at end of file
+locations: "AzureCloud:eastus2euap"
+owns_vm: true
+skip_on_clouds:
```

Comment:
Skip clouds meant for completely skipping test suite run(so that others will continue), it will apply for other clouds too depends on suite yml  definition.

Reply:
i'm not sure we have a use case for that, at least for now, but we can keep it

we should remove it from agent_update.yml, though. it is not needed and seems confusing to me

Reply:
it's needed, agent update won't work in china and usgov as rsm testing not possible there

Reply:
Since it's same runbook we share same suites in all clouds

Reply:
ok, i see. we discussed that the check for location would skip the cloud, but actually only skips the validation and you added a new property to skip the cloud. Sounds good.